appinventor/docs/reference/components/other.html

```diff
@@ -790,8 +790,8 @@ <h2 id="BarcodeScanner">
                     <img alt="Picture of Barcode Scanner component" src="images/barcodescanner.png" />
                     <p>
                       This non-visible component uses the phone's camera to read a 1-dimensional barcode
-                      or 2-dimensional barcode (QR code). In order for this component to work, the
-                      Barcode scanner app from ZXing must be installed on the phone. This app is
+                      or 2-dimensional barcode (QR code). If the UseExternalScanner property is toggled,
+                      the Barcode scanner app from ZXing must be installed on the phone. This app is
```

Comment:
I wouldn't say "toggled" here. I would say "set". Also change the reference to the Android Market to "Google Play" (which is what the market is called today).